docs/accessibility.md

```diff
@@ -0,0 +1,221 @@
+
+### Red and Green
+
+Red and green colors are not very noticeable for some types of color deficiency. For that reason, it is recommended to insert an accompanying icon, such as a checkmark (for success) or an exclamation point (for warning or error).
+
+For that task, `FontAwesome` is a supported choice.
+
+## Images
+
+### Image HTML tag
+
+If you add an image, make sure you also write an alternative text for it. The text should be descriptive and support the screen reader user.
+
+```jsx
+<img src={image} alt="A description of the image">
+```
+
+If you are not sure how to write this, please refer to [this guide](https://webaim.org/techniques/alttext/).
+
+### SVG
+
+For SVG images, do the following:
+
+- `<svg>` should have `aria-labelledby` attribute, with the ids of the following elements, separated by a blank space. It should also have a `role="img"`;
+- As the first child of `<svg>`, add `<title>` with an `id` attribute. The content of this element should be the title of the image;
+- (_Optional_) Following `<title>`, you can add `<desc>`, which describes the image. You should also add an `id`.
+
+```jsx
+<svg aria-labelledby="imageTitle imageDesc" role="img">
+  <title id="imageTitle"> A title of this image </title>
+  <desc id="imageDesc"> A description of this image </desc>
+  // ...
+</svg>
+```
+
+If your case is more specific, please check [this guide](https://css-tricks.com/accessible-svgs/#article-header-id-1).
+
+## Interactive elements
+
+When creating elements that have event listeners, prefer any component of the `reactstrap` interactive elements. Examples are: `Button`, `Input`, `DropdownToggle`. You can also choose a HTML `<a>` element.
+
+If you need to insert an event listener in a non-interactive element, such as a `span`, add also an `aria-role` of `button`, `link`, `checkbox`, or whatever seems closer to the functionality of the element.
+
+```html
+<span onClick="{someFunction}" aria-role="button">
+  Unconventional button
+</span>
+```
+
+### Dropdown menu
+
+There is a special case when you are creating a dropdown menu. First of all, try to follow [`reactstrap` structure](https://reactstrap.github.io/components/dropdowns/).
+
+Lastly, insert an additional tag `prop` to `DropdownItem` component.
+
+```jsx
+<DropdownItem tag="a"> Menu Item </DropdownItem>
+```
+
+## Forms, inputs and buttons
+
+Apart from inserting high color contrast combinations in buttons, also make sure it has a `:focus` style (usually a blue outline is enough). If you use `reactstrap`'s `Button` component, this is added by default, hence it is recommended.
+
+In case the content of the button is not a text (an icon or graphical element, for example), this element should have an `aria-label`, explaining what it does.
+
+```jsx
+<Button aria-label="Bookmark this" onClick={onClickFunction}>
```

Comment:
Even though this is just an example, we want to encourage people to be very explicit :) So perhaps "bookmark alert 12345" would be better.